examples/tutorial/00_Setup.ipynb

```diff
@@ -63,8 +65,32 @@
     "    ```\n",
     "    Note: You can replace \"lab\" with \"notebook\" if you prefer the original Jupyter notebook interface.\n",
     "\n",
-    "5. **Test HoloViz imports**:\n",
-    "After launching Jupyter, navigate to this notebook (`tutorial/00_Setup.ipynb`) and run the following cell. This command is a quick check that your environment is set up correctly. You should see the HoloViews, Bokeh, and Matplotlib logos along with a success message.\n"
+    "6. **Test HoloViz imports**:\n",
```

Comment:
<img width="881" alt="image" src="https://github.com/holoviz/holoviz/assets/35924738/ae15e45f-bf34-44df-96df-c23971dda077">

To avoid confusing users who follow the recommended path, I'd suggest instead moving the special steps 3, 4, 5 at the end.

Reply:
Agreed.